eng/pipelines/templates/stages/archetype-python-release.yml

```diff
@@ -271,3 +271,34 @@ stages:
               twine upload --repository $(DevFeedName) --config-file $(PYPIRC_PATH) $(Pipeline.Workspace)/${{parameters.ArtifactName}}/${{artifact.name}}/*-*a*.zip
               echo "Uploaded sdist to devops feed $(DevFeedName)"
             displayName: 'Publish ${{artifact.name}} alpha package'
+
+    - job: PublishDocsToNightlyBranch
```

Comment:
Do we want a `DependsOn: PublishPackages` @danieljurek has that in his .NET daily docs PR.

Reply:
It's missing from the JS release YAML but it would be a good idea to add that in. https://github.com/Azure/azure-sdk-for-js/blob/main/eng/pipelines/templates/stages/archetype-js-release.yml#L277 

I opened this PR for JS: https://github.com/Azure/azure-sdk-for-js/pull/17835